classes/class.ilAdobeConnectServer.php

```diff
@@ -341,6 +353,38 @@ public function getXUserId()
 		return $this->x_user_id;
 	}
 
+	/**
+	 * @return null
+	 */
+	public function getApiVersion()
+	{
+		return $this->api_version;
+	}
+
+	/**
+	 * @param null $api_version
+	 */
+	public function setApiVersion($api_version)
+	{
+		$this->api_version = $api_version;
+	}
+
+	/**
+	 * @return bool
+	 */
+	public function getHtmlClient()
+	{
+		return $this->html_client;
+	}
+
+	/**
+	 * @param bool $html_client
+	 */
+	public function setHtmlClient($html_client)
```

Comment:
`setHtmlClientUsage` or something similar, which indicates that a boolean value could be set with this mutator